Unix/build.mak

```diff
@@ -392,13 +392,20 @@ CHECKDIR=/tmp/omicheck.$(shell ./buildtool username)
 check:
 	rm -rf $(CHECKDIR)
 	( cd $(CONFIGUREDIR); $(OUTPUTDIR)/install --destdir=$(CHECKDIR) )
-	(  export LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
-           export DYLD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR); \
+	(  LD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR);  \
+	   export LD_LIBRARY_PATH;  \
+           DYLD_LIBRARY_PATH=$(CHECKDIR)/$(CONFIG_LIBDIR); \
```

Comment:
Different indentation from prior line? Also, for simplicity, why not just do `DYLD_LIBRARY_PATH=$LD_LIBRARY_PATH`  so a path change need only be done in one place? Also check indentation on line below.